src/wallet/api.rs

```diff
@@ -1144,7 +1144,14 @@ impl Wallet {
         &self,
         amount: Amount,
         feerate: f64,
+        manually_selected_outpoints: Option<Vec<OutPoint>>,
```

Comment:
Pass in a reference. `Option<&Vec<Outpoint>>`. This will reduce a bunch of cloning required in few places.